qiskit_nature/operators/second_quantization/particle_op.py

```diff
@@ -12,44 +12,48 @@
 
 """The Sum Operator base interface."""
 
-import copy
 from abc import ABC, abstractmethod
-from typing import Any, List
 
+from qiskit.opflow import PauliSumOp
 
-class SumOp(ABC):
-    """The Sum Operator base interface.
+from .star_algebra import StarAlgebraMixin
+from .tolerances import TolerancesMixin
+
+
+class ParticleOp(StarAlgebraMixin, TolerancesMixin, ABC):
+    """The Second Quantized Operator base interface.
 
     This interface should be implemented by all creation- and annihilation-type particle operators
     in the second-quantized formulation.
     """
 
-    def __init__(self, particle_type):
-        self._particle_type = particle_type
-
-    @property
-    def particle_type(self):
-        """Return the particle type"""
-        return copy.deepcopy(self._particle_type)
-
     @property
     @abstractmethod
     def register_length(self) -> int:
         """Getter for the length of the particle register that the SumOp acts on."""
         raise NotImplementedError
 
-    @property
     @abstractmethod
-    def operator_list(self) -> List[Any]:
-        """Getter for the operator_list of the `SumOp`"""
+    def to_opflow(self, method) -> PauliSumOp:
+        """TODO"""
         raise NotImplementedError
```

Comment:
Please remove this method from the interface and all subclasses.
We won't need it in this form.